homeassistant/components/tractive/__init__.py

```diff
@@ -210,6 +215,15 @@ async def _listen(self) -> None:
                     ):
                         self._last_pos_time = event["position"]["time"]
                         self._send_position_update(event)
+            except aiotractive.exceptions.UnauthorizedError:
+                self._config_entry.async_start_reauth(self._hass)
+                asyncio.create_task(self.unsubscribe())
+                _LOGGER.exception(
```

Comment:
```suggestion
                _LOGGER.error(
```

Do we need to log the whole exception backtrace here?  It seems we know why it happened

Reply:
I think this one is still not resolved.